WordPress/Classes/Services/BlogService.m

```diff
@@ -443,12 +444,23 @@ - (Blog *)findBlogWithXmlrpc:(NSString *)xmlrpc
 
 - (Blog *)createBlogWithAccount:(WPAccount *)account
 {
-    Blog *blog = [NSEntityDescription insertNewObjectForEntityForName:NSStringFromClass([Blog class])
+    NSString *entityName = NSStringFromClass([Blog class]);
+    Blog *blog = [NSEntityDescription insertNewObjectForEntityForName:entityName
                                                inManagedObjectContext:self.managedObjectContext];
     blog.account = account;
+    blog.settings = [self createSettingsWithBlog:blog];
     return blog;
 }
 
+- (BlogSettings *)createSettingsWithBlog:(Blog *)blog
+{
+    NSString *entityName = [BlogSettings classNameWithoutNamespaces];
```

Comment:
> As far as i can tell, the only scenario in which the classname might differ from the entity, is WPAccount.

For people management, the class name is `ManagedPerson`, since I'm using `Person` to detach the models from Core Data.